tests/test_customdb.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
```

Comment:
Not really needed IMO

Reply:
It is copied from the other test files. You want me to remove it from all files?

However, I fixed a typo in the name of the class `TestCustomMatrix`.

Reply:
Yeah pls remove it from all the files

Reply:
done